label_studio/data_export/api.py

```diff
@@ -154,8 +153,8 @@ def get(self, request, *args, **kwargs):
             )
         },
         extensions={
-            'x-fern-sdk-group-name': 'projects',
-            'x-fern-sdk-method-name': 'export',
+            'x-fern-sdk-group-name': ['projects', 'exports'],
+            'x-fern-sdk-method-name': 'download_sync',
```

Comment:
why this name?

Reply:
it's what we called easy export before: https://label-studio.docs.buildwithfern.com/api-reference/api-reference/projects/exports/download-sync